segmentstore/server/src/test/java/io/pravega/segmentstore/server/logs/ThrottlerTests.java

```diff
@@ -145,6 +157,187 @@ public void testInterruptedDurableDataLogDelay() throws Exception {
         testInterruptedDelay(ThrottlerCalculator.ThrottlerName.DurableDataLog);
     }
 
+    /**
```

Comment:
Working on reducing the amount of duplicate code within these tests. 

I tried to avoid importing metrics classes, however it seemed the most appropriate place, as the accurate reporting of the Metrics is entirely determined on their placement in the `Throttler` class.